suite-native/app/app.config.ts

```diff
@@ -169,6 +169,29 @@ export default ({ config }: ConfigContext): ExpoConfig => {
                 monochromeImage: './assets/appIcon_android.png',
                 ...appIconAndroid,
             },
+            intentFilters:
```

Comment:
If I understand well this configuration is only for Android, right?

Reply:
Yes this PR only supports Android, I would leave iOS for later, after the testing phase